migrations/11_create_nfts_metadata.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE nfts_metadata (
```

Comment:
can we rename to erc721_metadata?

Reply:
Yes. Can we rename the existing table of `nft_tokens` as well?

Reply:
ohhh, we already had one? 🤔 
then a lot of comments here need to reviewed